ParkingZoneApp/ViewModels/ParkingSlotViewModels/ListOfParkingSlotsVM.cs

```diff
@@ -24,7 +23,6 @@ public ListOfParkingSlotsVM()
         {
```

Comment:
let's make it beautiful anyway it is not in use
public ListOfParkingSlotsVM() {   }